react/ProgressionBanner/index.jsx

```diff
@@ -28,7 +27,7 @@ const ProgressionBanner = withStyles(styles)(
         <Banner
           className={classes.banner}
           icon={icon}
-          text={<Typography variant="h6">{text}</Typography>}
+          text={<span className="u-fz-medium u-fw-bold">{text}</span>}
```

Comment:
I don't think this is the right fix? We should rely on typography or at least to the theme defining the variant if needed. But I think that `<Typography variant="h6" component="span">` should work?

Reply:
pas bête je vais tester